src/App.css

```
@@ -923,6 +1013,496 @@ table.not-prose th {
+  background: var(--color-accent);
+  box-shadow: 0 0 0 1px color-mix(in srgb, var(--color-bg) 80%, transparent);
+  pointer-events: none;
+}
+
+.notion-table-announcement {
+  position: fixed;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)